projects/storefrontlib/src/cms-components/cart/add-to-wishlist/add-to-wish-list.component.html

```diff
@@ -11,7 +11,7 @@
             [disabled]="loading$ | async"
           >
             <cx-icon [type]="iconTypes.HEART"></cx-icon>
-            <span class="button-text">{{
+            <span class="button-text cx-action-link">{{
```

Comment:
Generally speaking I'd prefer the action link class on the button, as it's better related imho. We could have also avoided the `span` element I guess. I see you do that in other occasions in this PR, but not sure if there's a common pattern in the code base. I leave it to you to decide.

Reply:
@tobi-or-not-tobi indeed! and fixed, can you please re-approve.